paddle/fluid/operators/conv_op.cc

```diff
@@ -229,31 +229,13 @@ framework::OpKernelType ConvOp::GetExpectedKernelType(
 
 #ifdef PADDLE_WITH_MKLDNN
   if (this->CanMKLDNNBeUsed(ctx, input_data_type)) {
-    int customized_type_value =
-        (input_data_type == framework::DataTypeTrait<int8_t>::DataType() ||
-         input_data_type == framework::DataTypeTrait<uint8_t>::DataType())
-            ? OperatorWithKernel::IndicateVarDataType(ctx, "Filter") ==
-                      framework::DataTypeTrait<int8_t>::DataType()
-                  ? kConvMKLDNNINT8WS8
-                  : kConvMKLDNNINT8
-            : kConvMKLDNNFP32;
```

Comment:
It seems then that those variables are declared in conv_op.h:33 but never used anymore

Reply:
Thanks, I cleaned these variables by this PR: [[Opt Code] Clean conv_op useless variable](https://github.com/PaddlePaddle/Paddle/pull/47213)